ledger-tool/src/main.rs

```diff
@@ -2310,13 +2296,7 @@ fn main() {
                 let ending_slot = value_t_or_exit!(arg_matches, "ending_slot", Slot);
                 let target_db = PathBuf::from(value_t_or_exit!(arg_matches, "target_db", String));
 
-                let source = open_blockstore(
-                    &ledger_path,
-                    AccessType::Secondary,
-                    None,
```

Comment:
Oh, there's a handful of these `wal_recovery_mode` values that could change behavior as well. Just please confirm there is no issue with these subcommands starting to respect that flag.

Reply:
I believe there are no issues with this one. All of the old cases that ignored the `--wal-recovery-flag` had `None` hard-coded in for the `Option<BlockstoreRecoveryMode>` field. If the flag is not passed, then the following piece will evaluate to `None`:
```rust
    let wal_recovery_mode = matches
        .value_of("wal_recovery_mode")
        .map(BlockstoreRecoveryMode::from);
```
So, the only change of behavior would be the case where someone had been passing the flag on a command that ignored it. But, I'm of the mindset that if someone wants to go out of their way to set a flag then we should pass that flag along.